input.go

```diff
@@ -85,3 +89,21 @@ type ContractCallInput struct {
 	// The function must be public (e.g. in Iele `define public @functionName(...)`)
 	Function string
 }
+
+// GetCodeUpgrade gets the code upgrade from the arguments
+func (callInput *ContractCallInput) GetCodeUpgrade() ([]byte, error) {
+	if len(callInput.Arguments) < 1 {
+		return nil, ErrInvalidUpgradeArguments
+	}
+
+	return callInput.Arguments[0], nil
+}
+
+// GetCodeMetadataUpgrade gets the code metadata upgrade from the arguments
+func (callInput *ContractCallInput) GetCodeMetadataUpgrade() ([]byte, error) {
```

Comment:
I suggest replacing this method with `runtimeContext.GetCodeMetadataFromArgs()` inside Arwen.

Reply:
Removed from here, will be added on `runtimeContext`.